pkg/cmd/network/create.go

```diff
@@ -33,7 +33,7 @@ func Create(options types.NetworkCreateOptions, stdout io.Writer) error {
 		options.Subnets = []string{""}
 	}
 
-	e, err := netutil.NewCNIEnv(options.GOptions.CNIPath, options.GOptions.CNINetConfPath)
+	e, err := netutil.NewCNIEnv(options.GOptions.CNIPath, options.GOptions.CNINetConfPath, netutil.WithNamespace(options.GOptions.Namespace))
```

Comment:
Do we want to have an integration test to ensure that the created network is not leaked to other namespaces?

Reply:
Yeah, I add some in another dev-branch focused on test cleanup but I just took them out and added them here.